ppr-ui/src/views/userAccess/QsInformation.vue

```diff
@@ -1,18 +1,93 @@
 <template>
-  <span>Qualified Supplier Information Placeholder</span>
+  <div id="qs-information">
+    <section class="qs-information-intro mt-10">
+      <p>
+        To request <strong>{{ getMhrSubProduct }}</strong> access to the Manufactured Home Registry, complete the
+        information below. BC Registries staff will review your application and if approved, will provide access.
+      </p>
+      <CautionBox class="mt-8" :setMsg="`This is intended for the exclusive use of B.C. ${getMhrSubProduct} only.`" />
+    </section>
+
+    <section class="qs-service-agreement mt-8">
+      <h2>Service Agreement</h2>
+      <p class="mt-1">
+        Read and accept the terms of the service agreement. You may download and a copy of this agreement to save for
+        your records.
+      </p>
+      <v-btn outlined color="primary" class="mt-2" :ripple="false">
+        <img alt="" src="@/assets/svgs/pdf-icon-blue.svg" />
+        <span class="pl-1">Download Qualified Suppliers Agreement</span>
+      </v-btn>
+    </section>
+
+    <section class="qs-information-form mt-8">
+      <h2>Qualified Supplier ({{ getMhrSubProduct }}) Information</h2>
+      <p class="mt-1">
+        Provide the name and contact information for the Qualified Supplier.
+      </p>
+
+      <FormCard label="Qualified Supplier" :showErrors="showErrors" :class="{'border-error-left': showErrors}">
+        <template v-slot:formSlot>
+          <PartyForm
+            ref="qsInformationRef"
+            :baseParty="getMhrQsInformation"
+            :schema="PartyFormSchema"
+            :orgLookupConfig="orgLookupConfig"
+            @isValid="updateQsInfoValid"
+          >
+            <template v-slot:businessNameSlot>
+              <label class="generic-label" for="business-name">Qualified Supplier's Legal Business Name</label>
+              <p class="mt-2">
+                You can find the full legal name of an active B.C. business by entering the name or incorporation
+                number of the business, or you can type the full legal name of the Qualified Supplier if it is not a
+                registered B.C. business.
+              </p>
+            </template>
```

Comment:
@orelbn An example of the slot we created in use, basically think of slots as an override. 
In this context, we are overriding the basic template values and adding our own. 
If we didn't define this slot, it would default to what we have in the PartyForm Template. 

Real value here, not having to build out countless props and/or configuration for every single permutation in a template.
There is pros and cons to both.
This is more of a take-all-comers approach.